week1.ipynb

```diff
@@ -0,0 +1 @@
source":"**Ex. Problem 2:**\n\nNick and Ally were lounging on rafts in the shallow waters of the beach at Lake Bluebird. They were spaced 1.8 meters apart. A motor boat zoomed past creating ripples which traveled towards Nick and Ally. Nick and Ally's rafts began to bob up and down as the ripples passed by them, making exactly 4 up and down cycles in 8.4 seconds. When Nick's raft was at a high point, Ally's raft was at a low point and there were no crests between their boats. Determine the wavelength, frequency and speed of the ripples Assume that the ripples traveled in a direction parallel to the imaginary line connecting the two rafts.\n\n**Ex. Solution 2:**","metadata":{}},{"cell_type":"code","source":"# let's make a dictionary with known variables\nripple_vars = {\"distance btwn N & A\": 1.8, \"cycles\": 4, \"time\": 8.4}\nprint(ripple_vars)\n\n# first step: calculate the frequency (f)\nf = ripple_vars[\"cycles\"]/ripple_vars[\"time\"]\nprint(round(f,2), \"= frequency (Hz)\")\n\n# second step: calculate the wavelength (wl)\nwl = ripple_vars[\"distance btwn N & A\"]*2 # multiply by 2 since distance between N & A is half a wavelength (see prompt)\nprint(wl, \"= wavelength (m)\")\n\n# third and final step: calculate the speed (v)\nv = f * wl\nprint(round(v, 2), \"= speed (m/s)\")","metadata":{"trusted":true},"execution_count":null,"outputs":[]},{"cell_type":"markdown","source":"## Questions\n\nNow it's your turn to solve some wave fundamental problems on your own. Let's start by building off of Problem 1 above.\n\n**Goal:** \n\nCalculate the wavelength of the 1990 tsunami (see Problem 1). \n\n**Steps:**\n\n1) Convert travel time of the tsunami wave in hours to seconds. 1 hr = 3600 s. \n\n2) Calculate the frequency of the tsunami wave in Hz.\n\n3) Rearrange the wave equation to solve for $λ$.\n\n4) Your result should be pretty big. Convert your answer in meters back to miles. 1 m = 6.214E-4 miles.\n\n5) What do you notice about the wavelength? ","metadata":{}},{"cell_type":"code","source":"# your work for step 1 goes here","metadata":{"trusted":true},"execution_count":null,"outputs":[]},{"cell_type":"code","source":"# step 2","metadata":{"trusted":true},"execution_count":null,"outputs":[]},{"cell_type":"code","source":"# step 3","metadata":{"trusted":true},"execution_count":null,"outputs":[]},{"cell_type":"code","source":"# step 4","metadata":{"trusted":true},"execution_count":null,"outputs":[]},{"cell_type":"code","source":"# step 5","metadata":{"trusted":true},"execution_count":null,"outputs":[]},{"cell_type":"markdown","source":"## You've reached the end of the lesson. Great work!","metadata":{}},{"cell_type":"markdown","source":"## References\nhttps://www.sciencelearn.org.nz/resources/119-fundamentals-of-waves#:~:text=All%20kinds%20of%20waves%20have,%2C%20frequency%2C%20speed%20and%20amplitude.&text=A%20wave%20can%20be%20described,a%20disturbance%20that%20transfers%20energy.\n\nhttps://www.physicsclassroom.com/calcpad/waves","metadata":{}}]}
```

Comment:
<p>that looks great</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-aeckland2696/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>